_posts/2019-03-04-merge-time-review-complexity.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: "Merge time vs Change complexity in Foreman Core"
+author: "Greg Sutcliffe"
+date: "4 March 2019"
+tags:
+  - foreman
+  - data
+  - process
+  - community
+---
+
+In this post I'm going to dig into some of the data we have on GitHub about how
+our community functions. In particular, I'm looking to investigate some of the
+issues raised
+[here](https://community.theforeman.org/t/improving-pr-merge-velocity/12992) to
```

Comment:
stop. linking. to. "here". :)

`investigate some of the [issues raised on Discourse](…)`

Reply:
Sorry, bad habit. Fixed.